README.md

```diff
@@ -56,70 +57,68 @@ It's modeled after Python's `json.tool`, reading from stdin and writing to stdou
   * [Miscellaneous](#miscellaneous)
      * [Expanding `@optional` vertex fields](#expanding-optional-vertex-fields)
      * [Optional `type_equivalence_hints` compilation parameter](#optional-type_equivalence_hints-parameter)
+  * [FAQ](#faq)
   * [License](#license)
 
-## FAQ
-
-**Q: Does the compiler support all GraphQL language features?**
-
-A: No. We prioritized and implemented a subset of all functionality supported by the
-   GraphQL language. We hope to add more functionality over time.
-
-**Q: Do you really use GraphQL, or do you just use GraphQL-like syntax?**
-
-A: We really use GraphQL. Any query that the compiler will accept is entirely valid GraphQL,
-   and we actually use the Python port of the GraphQL core library for parsing and type checking.
-   However, since the database queries produced by compiling GraphQL are subject to the limitations
-   of the database system they run on, our execution model is somewhat different compared to
-   the one described in the standard GraphQL specification. See the
-   [Execution model](#execution-model) section for more details.
-
-**Q: Does this project come with a GraphQL server implementation?**
-
-A: No -- there are many existing frameworks for running a web server. We simply built a tool
-   that takes GraphQL query strings (and their parameters) and returns a query string you can
-   use with your database. The compiler does not execute the query string against the database,
-   nor does it deserialize the results. Therefore, it is agnostic to the choice of
-   server framework and database client library used.
-
-**Q: What databases and query languages does the compiler support?**
-
-A: We currently support a single graph database, OrientDB version 2.2.28+, and two query languages
-   that OrientDB supports: the OrientDB dialect of `gremlin`, and OrientDB's own custom SQL-like
-   query language that we refer to as `MATCH`, after the name of its graph traversal operator.
-   With OrientDB, `MATCH` should be the preferred choice for most users, since it tends to run
-   faster than `gremlin`, and has other desirable properties. See the
-   [Execution model](#execution-model) section for more details.
-   
-   Support for relational databases including PostgreSQL, MySQL, SQLite,
-   and Microsoft SQL Server is a work in progress. A subset of compiler features are available for
-   these databases. See the [SQL](#sql) section for more details.
+## Features
+* **Databases and Query Languages:** We currently support a single database, OrientDB version 2.2.28+, and two query languages that OrientDB supports: the OrientDB dialect of gremlin, and OrientDB's own custom SQL-like query language that we refer to as MATCH, after the name of its graph traversal operator. With OrientDB, MATCH should be the preferred choice for most users, since it tends to run faster than gremlin, and has other desirable properties. See the Execution model section for more details.
```

Comment:
The docs are a bit inconsistent. Here we say `We currently support a single database, OrientDB`, but later in the `SQL` section we say `Relational databases are supported by compiling to SQLAlchemy`. @jmeulemans should we update this section to include `SQL` as a backend, or do we wait for your branch to merge into master before we do that?

Reply:
There's an outstanding branch with the prototype SQL compiler, but that won't ever merge. The docs here are accurate, why are we removing the SQL details?

Reply:
I am sorry I deleted the SQL details by mistake. Let me fix that.